views/build/grunt/portableelement.js

```diff
@@ -153,99 +158,172 @@ module.exports = function (grunt) {
         return model;
     }
 
-    grunt.registerTask('portableelement', 'Compile Portable Elements', function () {
+    function transpile(sourceFilePath, sourceMapFilePath) {
+        if (!grunt.file.exists(sourceFilePath)) {
+            return Promise.reject(new Error(`Unable to find or read "${sourceFilePath}"`));
+        }
+        const code = grunt.file.read(sourceFilePath);
+
+        const hasSourceMap = grunt.file.exists(sourceMapFilePath);
+        const sourceMap = hasSourceMap ? grunt.file.read(sourceMapFilePath) : null;
 
-        var done = this.async();//async mode because requirejs optimization is an async process
-        var extension = grunt.option('extension') || grunt.option('e');
-        var selectedId = grunt.option('identifier') || grunt.option('i');
-        var manifests, compileTasks;
-        var self = this;
+        return babel
+            .transformAsync(code, {
+                presets: [
+                    [
+                        '@babel/env',
+                        {
+                            targets: 'extends @oat-sa/browserslist-config-tao',
+                            useBuiltIns: false
+                        }
+                    ],
+                    [
+                        'minify',
+                        {
+                            mangle: false
+                        }
+                    ]
+                ],
```

Comment:
Just a thought - would it make sense to keep common options in `.babelrc`?

Reply:
Yes that should the case usually if the bundling would be part of each PCI (it should be in my opinion) but since we rely on a central bundling now for any PCI, we provide the babel configuration manually.